src/device-state/current-state.ts

```diff
@@ -132,7 +132,7 @@ const getStateDiff = (): DeviceStatus => {
 
 const report = _.throttle(async () => {
 	const conf = await config.getMany([
-		'deviceId',
+		'deviceApiKey',
```

Comment:
Have we always reported the `deviceApiKey` to the cloud? That's strange

Reply:
Yeah I believe it's [needed](https://github.com/balena-os/balena-supervisor/blob/4eb571e992907512521f4e8e584433d44a34c67f/src/device-state/current-state.ts#L81) for the patch request to the backend, it was here before this PR too

Reply:
I understand is necessary as a header, but in the body? In any case I agree that is not related to this PR

Reply:
What do you mean? The only place deviceApiKey is used is in the [header](https://github.com/balena-os/balena-supervisor/blob/4eb571e992907512521f4e8e584433d44a34c67f/src/device-state/current-state.ts#L84)

Reply:
You are right, I was misreading the code